ibm/mas_devops/common_vars/casebundles/v8-230407-amd64.yml

```diff
@@ -0,0 +1,60 @@
+---
+# Case bundle configuration for IBM Maximo Operator Catalog v230407
+# -----------------------------------------------------------------------------
+# In the future this won't be necessary as we'll be able to mirror from the
+# catalog itself, but not everything in the catalog supports this yet (including MAS)
+# so we need to use the CASE bundle mirror process still.
+
+catalog_digest: sha256:a2120ecbe06ff90d4c9f3be6cd32b8ddf6a44796e96da8af4bd325338d2b6980
+
+# Dependencies
+# -----------------------------------------------------------------------------
+common_svcs_version: 1.19.0   # Operator version 3.23.0 (https://github.com/IBM/cloud-pak/tree/master/repo/case/ibm-cp-common-services)
+db2u_version: 4.5.3           # Operator version 2.2.0 (https://github.com/IBM/cloud-pak/tree/master/repo/case/ibm-db2uoperator)
+events_version: 4.4.0         # Operator version 4.4.0 (https://github.com/IBM/cloud-pak/tree/master/repo/case/ibm-events-operator)
+uds_version: 2.0.9            # Operator version 2.0.9
+sls_version: 3.6.0            # Operator version 3.6.0
+tsm_version: 1.5.0            # Operator version 1.5.0
+dd_version: 1.1.3             # Operator version 1.1.3
+
+# Maximo Application Suite
+# -----------------------------------------------------------------------------
+mas_core_version:
+  8.9.x: 8.9.3    # No Update
+  8.10.x: 8.10.0  # Updated
+mas_assist_version:
+  8.9.x: 8.6.2    # No Update
+  8.10.x: 8.7.0   # Updated
+mas_hputilities_version:
+  8.9.x: 8.5.1    # No Update
+  8.10.x: 8.6.0   # Updated
+mas_iot_version:
+  8.9.x: 8.6.4    # No Update
+  8.10.x: 8.7.0   # Updated
+mas_manage_version:
+  8.9.x: 8.5.3    # No Update
+  8.10.x: 8.6.0   # Updated
+mas_monitor_version:
+  8.9.x: 8.9.4    # No Update
+  8.10.x: 8.10.0  # Updated
+mas_optimizer_version:
+  8.9.x: 8.3.2    # No Update
+  8.10.x: 8.4.0   # Updated
+mas_predict_version:
+  8.9.x: 8.7.1   # No Update
+  8.10.x: 8.8.0   # Updated
+mas_visualinspection_version:
+  8.9.x: 8.7.0   # No Update
+  8.10.x: 8.8.0   # Updated
+
+# Extra Images for UDS
+# ------------------------------------------------------------------------------
+uds_extras_version: 1.2.0
+
+# Extra Images for Mongo
+# ------------------------------------------------------------------------------
+mongo_extras_version: 4.2.23
+
+# Extra Images for Db2u
+# ------------------------------------------------------------------------------
+db2u_extras_version: 1.0.1
```

Comment:
We need to update this for v3.1 channel that we are going to start using now (for OCP 4.12 compatability)